doc/source/user_guide/ug_implementation.rst

```diff
@@ -1402,15 +1402,21 @@ Project (SIMIP) :cite:`Notz16` are available as history output variables
 `monthly <http://clipc-services.ceda.ac.uk/dreq/u/MIPtable::SImon.html>`_ and
 `daily <http://clipc-services.ceda.ac.uk/dreq/u/MIPtable::SIday.html>`_
 requested SIMIP variables provide their history field names in CICE.
-These variables have been updated for the  `CMIP7 data request <https://wcrp-cmip.org/cmip7-data-request-v1-0/>`_.
+These variables have been updated for the  
+`CMIP7 data request <https://wcrp-cmip.org/cmip7-data-request-v1-0/>`_.
 
 The ``f_CMIP`` flag has been removed. This is now a ``set_nml.cmip`` namelist option
 which can be invoked with the ``-s cmip`` option during cice.setup. This optional
-namelist setting will turn on the CMIP data request and turn of CICE duplicates of
+namelist setting will turn on the CMIP data request and turn off CICE duplicates of
 SIMIP variables. However, these can be changed by the user in their case ``ice_in`` file.
 Note that all SIMIP variables have been updated to correspond to the new 
 `CMIP7 data request <https://wcrp-cmip.org/cmip7-data-request-v1-0/>`_. 
 
+Note that some SIMIP variables require division ice or sub=ice areas, which can be extremely
```

Comment:
2 typos:  "division by ice or sub-ice areas"

Also, please enclose "such as ``sithick``" either in commas or parentheses.